index.js

```diff
@@ -205,7 +211,11 @@ var TruffleContractSchema = {
     // update schema version
     normalized.schemaVersion = pkgVersion;
 
-    return normalized;
+    if (options.validate) {
```

Comment:
I mean the link references inside bytecode... because solc puts path delimiter characters inside the underscore block, and you've talked about standardizing what that looks like.

I would think the correct answer is to perform normalization of the unlinked bytecode itself, but not sure when we should do that.